src/components/TheHeader.vue

```diff
@@ -1,11 +1,28 @@
 <template>
   <header class="header">
     <div class="logo">
-      <img src="@/assets/img/logo.svg" />
+      <router-link to="/">
+        <img src="@/assets/img/logo.svg" />
+      </router-link>
     </div>
-    <div class="header__buttons">
-      <MainButton button-type="secondary">Register</MainButton>
-      <MainButton button-type="primary">Login</MainButton>
+    <div class="header__navbar" v-if="!isLogin && !isRegister">
```

Comment:
maybe instead of three `v-if`s it would be easier to wrap the whole thing with some "fragment" (which is called `<template>` in vue 😉) and just do `v-if`, `v-else`?

Reply:
That is amazing! I wasn't sure how to make fragments in Vue, thank you as this reads so much better in my head :)